Makefile.PL

```diff
@@ -1,8 +1,35 @@
-use ExtUtils::MakeMaker;
+use ExtUtils::MakeMaker 6.64;
 WriteMakefile(
     NAME         => 'Number::Compare',
     VERSION_FROM => 'lib/Number/Compare.pm',
+
     PREREQ_PM => {
+       'Carp' => 0,
+       'strict' => 0,
+       'warnings' => 0,
+    },
+
+    TEST_REQUIRES => {
        'Test::More' => 0,
-   },
+    },
+
+    CONFIGURE_REQUIRES => {
+        "ExtUtils::MakeMaker" => 6.64,
+    },
+
+    LICENSE => 'perl',
+
+    MIN_PERL_VERSION => 5.006,
+
+    META_MERGE => {
+        'meta-spec' => { version => 2 },
+        resources => {
+            repository  => {
+                type => 'git',
+                web  => 'https://github.com/richardc/perl-number-compare',
+                url  => 'https://github.com/richardc/perl-number-compare.git',
+            },
```

Comment:
Where can I read more about what these are? I want to understand the contract that's being established here.

Reply:
The official place is [CPAN::Meta::Spec](https://metacpan.org/pod/CPAN::Meta::Spec), but in summary
* the META_MERGE is setting adding the github repo to the CPAN metadata. With that, MetaCPAN will put a link in the sidebar
* the LICENSE key is so the metadata specifies the license to match what the doc says. Various linux distro builders like to have the license in the metadata, and won't include a CPAN distribution if it doesn't have it
* The distribution metadata can now classify prereqs according to whether they're needed for configuration, runtime, test, etc.